daemon/k8s_watcher.go

```diff
@@ -1632,3 +1675,56 @@ func (d *Daemon) deleteK8sPodV1(pod *v1.Pod) {
 	}
 
 }
+
+func (d *Daemon) addK8sNodeV1(k8sNode *v1.Node) {
+	logger := log.WithFields(logrus.Fields{
+		"K8sNodeName": k8sNode.ObjectMeta.Name,
+	})
+
+	n := k8s.ParseNode(k8sNode)
+	var hostIP net.IP
+	if n.Name != node.GetName() {
```

Comment:
Any reason to continue for the own node? Leaving `hostIP` nil seems weird here

Reply:
fixed

Reply:
Is this fixed? hostIP is still nil for the own node and nil is passed into `To4()`.

Reply:
@tgraf it is, but it was wrongly on the next commit. I've put the changes into this commit